torch_tvm/memory_utils.cpp

```diff
@@ -0,0 +1,51 @@
+#include "memory_utils.h"
+
+namespace torch_tvm {
+namespace utils {
+
+bool is_aligned(void* data_ptr, std::uintptr_t alignment_in_bytes) {
+  auto mask = alignment_in_bytes - 1;
+  AT_CHECK((alignment_in_bytes & mask) == 0);
```

Comment:
might be nice to make this a compile time check

Reply:
Well if we constexpr alignment to 64 or some value then perhaps we can convert this to static_assert. Then rename this to is_64B_aligned. But the alignment requirement comes from tvm so even though we know it needs 64B aligned, making it constexpr here will decouple it from tvm. Unfortunately due to some include issues I had to hardcode that value to 64 in some places, but that I intend to fix.

Reply:
You can make `alignment_in_bytes` a template argument and use static_assert.

Reply:
Only if kAllocAlignment value which comes from tvm is constexpr. I will check and see.